Mathlib/GroupTheory/Subgroup/Finite.lean

```diff
@@ -173,6 +173,12 @@ theorem card_le_one_iff_eq_bot [Fintype H] : Fintype.card H ≤ 1 ↔ H = ⊥ :=
 #align subgroup.card_le_one_iff_eq_bot Subgroup.card_le_one_iff_eq_bot
 #align add_subgroup.card_nonpos_iff_eq_bot AddSubgroup.card_le_one_iff_eq_bot
 
+@[to_additive] lemma eq_bot_iff_card [Fintype H] : H = ⊥ ↔ Fintype.card H = 1 := by
+  constructor
+  · rintro rfl
+    exact card_bot
+  · apply eq_bot_of_card_eq
```

Comment:
A shorter proof would be to `rw` with `Subgroup.card_eq_one` and `Nat.card_eq_fintype_card`.

Reply:
I like the fact that this lemma is in `Mathlib.GroupTheory.Subgroup.Finite` instead of `Mathlib.GroupTheory.Index`. And the above proof is already pretty short. But I compressed it as a compromise.

Reply:
Sure, I guess it does fit in more naturally here.